test/models/report_test.rb

```diff
@@ -0,0 +1,36 @@
+require "test_helper"
+
+class ReportTest < ActionController::TestCase
+  test "valid? returns false when report is too short" do
+    msg = "Report is too short (minimum is 100 characters)"
+    report = Report.new
+
+    assert_not report.valid?
```

Comment:
This is just a style suggestion, not really that important and maybe you prefer the current, but we could use a positive assertion: `assert report.invalid?`